packages/type-safe-api/src/project/type-safe-api-project.ts

```diff
@@ -166,7 +166,15 @@ export class TypeSafeApiProject extends Project {
     super(options);
 
     const nxWorkspace = this.getNxWorkspace(options);
-    const isNxTsWorkspace = this.parent instanceof NxMonorepoProject;
+
+    // instanceof will return false if the version of nx-monorepo the user has installed does not exactly match the one type-safe-api depends on,
+    // so we fall back to check that the parent looks like the typescript nx-monorepo.
+    const isNxTsWorkspace =
```

Comment:
Just wandering if we could do this instead?

`const isNxTsWorkspace = (<any>this.parent).constructor.name === NxMonorepoProject.name`

Reply:
Yeah I think that's cleaner and should work! Will update :)